### PR TITLE
UPGRADE file: Tips on 1.0.0 update

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | partially #8642 |
| License         | MIT |

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -37,13 +38,32 @@
...
 
+* Model classes now return ArrayCollections instead of arrays, so if you want to mock it in PHPSpec, 
+  you need to wrap mocked object like this:
+
+    ```php
+    function it_does_something(
+         ReviewableInterface $reviewable,
+         ReviewInterface $review
+    ): void {
+         $reviewable->getReviews()->willReturn(new ArrayCollection([$review>getWrappedObject()]));
```
Comment (Zales0123) [inline UPGRADE.md]: `$review>getWrappedObject()` -> `$review->getWrappedObject()`

File: UPGRADE.md [hunk 2]
```diff
@@ -37,13 +38,34 @@
   * Change autoload path in `phpunit.xml.dist`: replace 'app' with 'vendor'
   * Remove `"Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",` from composer.json scripts
 
+* Model classes now return ArrayCollections instead of arrays, so if you want to mock it in PHPSpec, 
+  you need to wrap mocked object like this:
```
Comment (venyii) [inline UPGRADE.md]: **the** mocked object

File: UPGRADE.md [hunk 3]
```diff
@@ -71,7 +93,7 @@
 
 * `ChannelFactory` has been made final, use decoration instead of extending it.
 
-* `ChannelAwareInterface::setChannel` does not longer have a default null argument and requires one to be explicitly passed.
+* `ChannelAwareInterface::setChannel` does no longer has a default null argument and requires one to be explicitly passed.
```
Comment (venyii) [inline UPGRADE.md]: ~~has~~ have
or remove the `does`

File: UPGRADE.md [hunk 4]
```diff
@@ -238,8 +260,8 @@
 ### Shipping / ShippingBundle
 
 * `UnresolvedDefaultShippingMethodException` has been made final, use decoration instead of extending it.
-* `setShippable(?ShippableInterface $shippable)` has been added to `ShipmentUnitInterface`.
-* The following methods does not longer have a default null argument and requires one to be explicitly passed:
+* `setShippable(?ShippableInterface $shippable)` method has been added to `ShipmentUnitInterface`.
+* The following methods no have a default null argument and require one to be explicitly passed:
```
Comment (venyii) [inline UPGRADE.md]: **don't** have
or
no **longer**

File: UPGRADE.md [hunk 5]
```diff
@@ -261,7 +283,7 @@
 
 ### ThemeBundle
 
-* `ThemeHierarchyProviderInterface::getThemeHierarchy` does not longer accepts null as the passed argument.
+* `ThemeHierarchyProviderInterface::getThemeHierarchy` does no longer accepts null as the passed argument.
```
Comment (venyii) [inline UPGRADE.md]: accept~~s~~

Comment (CoderMaggie) [reply UPGRADE.md]: or `no longer accepts` am I right @venyii ? :)

**Comment by lchrusciel** [comment]
Thanks Magda! 👍

**Pull request merged**